### ISSUE TITLE
quest controllers act like 3dof instead of 6dof

### ISSUE THREAD

**Issue by Rotaryknight** [GPUOpen-LibrariesAndSDKs/Radeon-ReLive-VR#16]
before the latest quest update everything was working fine. Controller wasnt shown after quest update so I updated relivevr from 1.0.12 to 1.0.14 and now the controller is shown and it shows on steamvr as connected but it only goes up and down and no x or y movement. There is Z movement and rotational and thats it. The buttons all do as it is suppose to do just no x and y movement.

**Comment by GennadiyAMD** [comment]
What is the version of the Oculus firmware that your device is showing after the update? Do you see both controllers or just one? Does it exhibit the "elbow model" behavior like tilting upwards when you raise it?


-------- Original Message --------

**Comment by Rotaryknight** [comment]
i am on version 7.0.0.32.358.163419639 and OS version 333700.3780.0
Both controllers shows up and it does not do the elbow tilt. it just rises or lowers where ever my center spot is.

**Comment by GennadiyAMD** [comment]
@Rotaryknight - I'm on the same versions and controllers are tracking fine. The only thing that I can think of is that the update didn't finish completely and your headset is stuck in some semi-updated state. Please try uninstalling the app completely with `adb uninstall com.amd.wirelesshmd` first and then install it again. Restart the headset after that.

Please let me know if this helps.

**Comment by Rotaryknight** [comment]
I uninstalled using the adb, restarted, reinstalled, and restarted and its the same thing. This time though I tried the same installation on my laptop and it works like normal...So I am guessing there is something wrong with my desktop with steam VR.

**Comment by GennadiyAMD** [comment]
@Rotaryknight - try running room setup on your desktop with your Quest connected. Standing only, as room scale wouldn't be able to run without lighthouses. If this doesn't help, try reinstalling Steam.

**Comment by Rotaryknight** [comment]
I ran the room setup and uninstalled in combination of deleting steamvr folder too. Still the same thing.  Its weird because it was working flawlessly before the quest update. but yet it works on my laptop. The only difference was probably because my laptop was in beta for steamvr and my desktop was not. I will try that out later tonight.

**Comment by Rotaryknight** [comment]
ok, so I figured out what was wrong, and it was by accident too. It was because the room was too dark....which doesnt make sense to me because the quest works fine in the same lighting condition. I turned on my lamp to do some soldering on a project, I forget to turn the lamp off, I put the quest on to fix this 3dof problem and it was fixed. I turned the light off and I am back to 3dof.....this is the weirdest problem I had so far. And it looks like my laptop worked because its in my living room and its a really lit room.

SO I guess its fixed lol
